app.js

```diff
@@ -204,12 +204,13 @@ mongoose
     logger.error(err);
   });
 
+const port = 1963;
 const databaseIssue = async () => {
   app.listen(4000, "0.0.0.0", ()=> {
```

Comment:
Hmm merge whoopsie: db is set to 1963 locally as I run multiple non-docker versions on the same PC.